src/challonge.service.js

```diff
@@ -1,121 +1,119 @@
 const R = require('ramda');
 
-function challongeServiceFactory({ api, organization }) {
-    async function fetchAllTournaments() {
-        const { data } = await api.get('tournaments.json', {
-            params: {
-                subdomain: organization,
-            },
-        });
-        return R.map(({ tournament }) => tournament)(data);
-    }
+const fetchAllTournaments = ({ api, organization }) => async function () {
+    const { data } = await api.get('tournaments.json', {
+        params: {
+            subdomain: organization,
+        },
+    });
+    return R.map(({ tournament }) => tournament)(data);
+}
```

Comment:
I am missing semicolons here and for all other service methods. :wink:

Reply:
Sorry, I need to get the same linter or something...

Reply:
I don't use any linter yet, apart from my mental one. 🙄